### PR TITLE
Project ready for UI design

### PR DESCRIPTION
**Pull request by mariam-rivera**
# Description
We already done the project setup with navigation through 2 screens (Log In and Sign In). Also we used the extension Prettier for get a structured and clean code. 
 - Two screens working successfully.
 - This project is ready for UI design. 

  ## Screenshots 
image
image
image
image

### CODE REVIEW

File: ReservacionDeVuelos/src/screens/LogInScreen.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {View, Text, StyleSheet, TouchableOpacity} from 'react-native';
+
+const LogInScreen = ({navigation}) => {
+  return (
+    <View style={styles.container}>
+      <Text style={styles.text}>LOG IN SCREEN</Text>
+      <TouchableOpacity onPress={() => navigation.navigate('SignIn')}>
+        <Text style={styles.goToSignIn}>
+          Don't have an account? Create here!
```
Comment (LeyaimMagma) [inline ReservacionDeVuelos/src/screens/LogInScreen.js]: could you create a constants.js file, that is to add the text LOG IN SCREEN and Don't have an account? Create here!, so you can import it wherever you need it. 

example constants.js
const LOGIN_SCREEN = 'LOG IN SCREEN'; 
CONST NO_ACCOUNT = 'Don't have an account? Create here!'

File: ReservacionDeVuelos/src/screens/SignInScreen.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  text: {
+    fontSize: 20,
+    color: 'black',
```
Comment (LeyaimMagma) [inline ReservacionDeVuelos/src/screens/SignInScreen.js]: Add a colors.js file to add all the colors that you will need

**Review by LeyaimMagma — CHANGES_REQUESTED** [review]

File: ReservacionDeVuelos/src/screens/SignInScreen.js (hunk 2 above)
Comment (mariam-rivera) [reply ReservacionDeVuelos/src/screens/SignInScreen.js]: Thanks, we will work on it!

**Review by Ing-Heber — DISMISSED** [review]
Nice job, works fine, please apply what leyaim told you and the tip I gave you

**Review by Ing-Heber — DISMISSED** [review]
Wonderful!!

**Review by Ing-Heber — DISMISSED** [review]

**Review by ChantalRamirez — APPROVED** [review]
It´s ok, nice

**Review by ChantalRamirez — DISMISSED** [review]
It is ok

File: ReservacionDeVuelos/App.js [hunk 3]
```diff
@@ -1,114 +1,8 @@
...
-    paddingRight: 12,
-    textAlign: 'right',
-  },
-});
+const App = () => {
+  return <Providers/>;
+}
 
-export default App;
+export default App;
```
Comment (LeyaimMagma) [inline ReservacionDeVuelos/App.js]: Add blank space here

**Review by LeyaimMagma — CHANGES_REQUESTED** [review]

**Review by GracielaJazmin — APPROVED** [review]
Good!

**Review by hearagmu — APPROVED** [review]
Vientos.

**Review by LeyaimMagma — APPROVED** [review]

**Review by LeyaimMagma — APPROVED** [review]

**Review by Ing-Heber — APPROVED** [review]

**Pull request merged**